closure/goog/testing/asserts_test.js

```diff
@@ -436,9 +438,61 @@ function testAssertObjectEqualsMap() {
   assertObjectEquals('maps should be equal', map1, map2);
 
   map1.set('hi', 'hey');
-  assertThrows('sets should not be equal', function() {
-    assertObjectEquals(map1, map2);
-  });
+  assertThrows('maps should not be equal',
+               function() { assertObjectEquals(map1, map2); });
+}
+
+
+function testAssertObjectEqualsTypedArray() {
+  var typedArrays = [
```

Comment:
Please use `let`, `const`, and arrow functions throughout this test case, as appropriate.

Reply:
Okay, will do.  (It's my preference... I just got confused by the actual code which seemed to require ES5.)